src/client/src/App.js

```diff
@@ -55,12 +61,17 @@ class App extends Component {
                     type="text"
                     value={this.state.text}
                     onKeyPress={this.onKeyPress}
-                    onChange={e => this.setState({ text: e.target.value })}/>
+                    onChange={e => this.setState({ text: e.target.value },()=> console.log())}/>
```

Comment:
Empty console log?

Reply:
lol fixed

Reply:
Perhaps some  of your changes aren't committed yet?